integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -169,13 +169,19 @@ def get_low_hix_value_context(self) -> dict[str, QuerySet]:
         if not settings.TEXTLAB_API_ENABLED or not self.request.region.hix_enabled:
             return {}
 
-        translations_under_hix_threshold = PageTranslation.objects.filter(
+        translations = PageTranslation.objects.filter(
             language__slug__in=settings.TEXTLAB_API_LANGUAGES,
             id__in=self.latest_version_ids,
             page__hix_ignore=False,
             hix_score__lt=settings.HIX_REQUIRED_FOR_MT,
         )
 
+        translations_under_hix_threshold = [
```

Comment:
Why do we do this calculation here? To me it's doing the exact same thing as before :thinking:

Reply:
`hix_score__lt=settings.HIX_REQUIRED_FOR_MT` collects translation with *raw* HIX score < threshold, but `not translation.hix_sufficient_for_mt` in the next step checks whether *rounded* HIX score satisfies the threshold or not. 

Both `rounded_hix_score` and `hix_sufficient_for_mt` are cached_property and cannot used in the query, but we wanted to avoid checking all the translation by loop. So the code is first filtering translations out that definetely have good HIX score and checks the edge cases of up-rounding after that.